lib/msf/ui/tip.rb

```diff
@@ -0,0 +1,38 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Module that contains some most excellent tips.
+    #
+    ###
+    module Tip
+      def self.highlight(string)
+        "%grn#{string}%clr"
```

Comment:
Where is the Rex-Text `substitute_colors` called? I'm not seeing it anywhere in the changes

Reply:
It should already be handled by the console because `print_line` is called above.